lib/app.dart

```diff
@@ -42,6 +43,7 @@ class _LemonadeAppViewState extends State<LemonadeApp> {
   @override
   void initState() {
     super.initState();
+    FlutterAppBadger.removeBadge();
```

Comment:
it keep this simple logic ? thought we have to like read the notification, the we removeBadge ?